ui/components/component-library/avatar-base/avatar-base.types.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  BackgroundColor,
+  BorderColor,
+  TextColor,
+} from '../../../helpers/constants/design-system';
+
+export enum AvatarBaseSizes {
+  XS = 'xs',
+  SM = 'sm',
+  MD = 'md',
+  LG = 'lg',
+  XL = 'xl',
```

Comment:
Can you change to Xs  to be consistent with what have been done in Icon?
https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/component-library/icon/icon.types.ts#L4
Thank you !

```suggestion
  Xs = 'xs',
  Sm = 'sm',
  Md = 'md',
  Lg = 'lg',
  Xl = 'xl',
```